content/30.conclusions.md

```diff
@@ -1,8 +1,9 @@
 ## Conclusions
 
-The presence of female STEM role models is associated with higher persistence for undergraduate women in geoscience [@doi:10.1130/GES01659.1].
+The underrepresentation of women as corresponding authors observed here is also pervasive in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956].
+In addition to holding fewer corresponding authorship positions, female scientists are less often to be cited [@arxiv:2001.01002], invited by journals to submit papers [@doi:10.1371/journal.pbio.2004956] and suggested as reviewers [@doi:10.1038/541455a].
```

Comment:
```suggestion
In addition to holding fewer corresponding authorship positions, female scientists are cited less often [@arxiv:2001.01002], invited by journals to submit papers less often [@doi:10.1371/journal.pbio.2004956] and suggested as reviewers less often [@doi:10.1038/541455a].
```
I realize it said the same thing before but for some reason my brain wasn't linking "less often" to the later portions. Hopefully this will prevent any confusion if that happens to someone else.

Reply:
I like the emphasis!